sBTC SIP

```diff
@@ -0,0 +1,166 @@
+
+**Six Sortitions for Withdrawal:** The sBTC Bootstrap signers will only trigger the withdrawal on Bitcoin after the Stacks withdrawal has been confirmed by 6 sortitions. Bitcoin transactions cannot be constructed such that they’re valid for only a specific Bitcoin fork, nor can they be forcefully taken back after they’re created, and a Bitcoin fork can cause a Stacks reorg such that the initial withdrawal request becomes invalid. The sBTC Bootstrap Signers will need to wait for the withdrawals to be sufficiently final in order to make the withdrawal on Bitcoin without risk of allowing a double withdrawal from the Stacks Blockchain. The withdrawal procedure will need to track the number of valid sortitions after the withdrawal to verify that the Stacks transaction is final, and 6 is the standard choice for Bitcoin finality.
+
+#### Withdrawal Flow
+The main steps of the sBTC withdrawal flow are as follows:
+1. **Withdrawal request:** An sBTC holder calls the withdraw-request function in the .sbtc contract.
+    - This transfers the requested amount of sBTC to the .sbtc contract & mints the user a non-transferable locked-sBTC as a placeholder.
+2. **Withdrawal accept:** If accepted, the following happens:
+    - The signers create a "Withdrawal accept" transaction on Bitcoin which returns the requested amount to the designated address.
+    - Once the Bitcoin transaction is confirmed, the signers must then call into the sBTC protocol to mark it as complete - this is done through a Stacks transaction.
+    - If successful, the resulting Stacks transaction will record the withdrawal request as complete & will accordingly burn the user’s locked-sBTC.
+3. **Withdrawal reject:** If instead the request is rejected, the sBTC signers will call the withdraw-reject function in the .sbtc smart contract. This function does the following:
+    - Returns the sBTC to the holder.
+    - Records the signer votes.
+
+To support deposits and withdrawals in sBTC, Signers must collectively make smart contract calls on Stacks. We require each signer to individually sign any contract call and ensure at least 70% of the signer signatures are present for each contract call to be valid.
+
+## Auxiliary Features
+Auxiliary features of the sBTC Bootstrap protocol are described below.
+
+- **Stacks Transaction Fee Sponsorship:** sBTC will include the option to have sBTC transactions on Stacks be sponsored in return for some sBTC. Using the approach suggested in this issue, sBTC users will be able to nearly spend sBTC as gas by getting support from an existing STX holder.
+- **Signer Wallet Rotation:** Mechanisms are provided for the scenario where a signer wants to rotate their key. For this to happen, signers must coordinate offline and vote on-chain on the new signer set (aka set of keys). Once the new signer set is determined, the signers conduct a wallet handoff and re-execute DKG.
```

Comment:
There are lots of ways this could fail, which this SIP must account for.  Here are a few (non-exhaustive) pain points that this sentence elides:

* DKG itself can fail due to a faulty coordinator
* The transaction to re-key can fail due to a low transaction fee (what if the signers don't have enough STX?)
* How does the contract validate the new signer sets' keys, if (presumably) some of the existing keys are compromised by malicious parties.  Like, is rekeying only supported if at least 70% of the keys remain honest?
* Do keys expire?
* How do signers agree on the new BTC address, given that some existing signer keys could be malicious?
* How is the sBTC contract kept in-sync with the new BTC address?  Do the new signers have to somehow write the new address to the contract, or is the contract even involved here?
* How does the API (or whatever means the user has to discover the deposit address) learn the new address?